selfdrive/controls/lib/events.py

```diff
@@ -135,8 +135,8 @@ def __str__(self) -> str:
     return f"{self.alert_text_1}/{self.alert_text_2} {self.priority} {self.visual_alert} {self.audible_alert}"
 
   def __gt__(self, alert2) -> bool:
-    return self.priority > alert2.priority
-
+    gt: bool = self.priority > alert2.priority
```

Comment:
Do you really need this variable here? What other type would a compare be?

Reply:
Oh, could use the same strategy as in bootlog file, will fix. Compare can generally be of type Any if the types aren't specified because __lt__ can be overloaded.